src/esm_parser/esm_parser.py

```diff
@@ -1133,7 +1134,7 @@ def merge_dicts(*dict_args):
     -------
     A merged dictionary (shallow).
     """
-    result = {}
+    result = {} #DictWithProvenance({}, {"provenance": None})
```

Comment:
```suggestion
    result = {}
```

Reply:
Keep it clean ;-) You can always go back into the git history if you need that again. Basically this code does not change...